src/gateway/tasks/verifyInteractions.ts

```diff
@@ -244,7 +255,7 @@ async function verifyInteractions(context: Application.BaseContext) {
         } else {
           logger.warn("Different response from peers for", {
             current_peer: statusesRounds[j][i],
-            prev_peer: statusesRounds[j - 1][i]
+            prev_peer: statusesRounds[j - 1][i] // Will it work correctly for j == 0?
```

Comment:
obviously it would not work properly for j = 0, but if we're in this branch (which means that we have different responses from rounds) than j >= 1.
i'll add a comment in code.